### ISSUE TITLE
#1342482653------------- Elements must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1339]
<strong>Source:</strong>`<span class="c-coordinateSummaryPanel__summaryStatsNum c-coordinateSummaryPanel__summaryStatsNum--blue t-jobSummaryTotal1">64</span>`

<strong>Element Location: </strong>.t-jobSummaryTotal1

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix any of the following:
  Element has insufficient color contrast of 4.14 (foreground color: #007ac2, background color: #f3f3f3, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

<strong>Tags:</strong> cat.color,wcag2aa,wcag143,TTv5,TT13.c,EN-301-549,EN-9.1.4.3,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.8/color-contrast?application=axeAPI
